src/components/Select/index.scss

```diff
@@ -10,10 +10,10 @@
   font-family: $font-family-primary !important;
   &__control,
   div[class*="-control"] {
-    background-color: transparent;
+    background-color: transparent !important;
     border-radius: $border-radius;
-    border-width: 1px;
-    border-color: $input-borderColor--inactive;
+    border-width: 1px !important;
+    border-color: $input-borderColor--inactive !important;
```

Comment:
add !important because previously it can't override the default style. only happens when we import Select directly. we don't need to add !important if we import ValidSelect